eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -121,10 +119,117 @@ function Confirm-ChangeLogEntry {
   return $true
 }
 
-function Set-TestChangeLog($TestVersion, $changeLogFile, $ReleaseEntry) {
-  Set-Content -Path $changeLogFile -Value @"
-# Release History
-## $TestVersion ($(Get-Date -f "yyyy-MM-dd"))
-- $ReleaseEntry
-"@
+function New-ChangeLogEntry {
+  param (
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [String]$Version,
+    [String]$Status="(Unreleased)",
+    [String]$Title,
+    [String[]]$Content
+  )
+
+  # Validate relase Status
+  $Status = $Status.Trim().Trim("()")
+  if ($Status -ne "Unreleased") {
+    $dateFormat = "yyyy-MM-dd"
+    $provider = [System.Globalization.CultureInfo]::InvariantCulture
+    try {
+      $Status = ([System.DateTime]::ParseExact($Status, $dateFormat, $provider)).ToString($dateFormat)
```

Comment:
Is there a reason that doing something like `[DateTime]$status` won't work?

Reply:
Assigning the string directly like this [DateTime]$Status= 2019-15-6 does not result in the correct datetime.